src/collectd.conf.pod

```diff
@@ -5809,6 +5810,13 @@ Dispatch the OVS DB interface link status value with configured plugin interval.
 Defaults to false. Please note, if B<SendNotification> and B<DispatchValues>
 options are false, no OVS information will be provided by the plugin.
 
+=item B<NotifyInterfaceAdd> I<true|false>
+
+When a new interface is added, its status typically changes from unset to down.
+This leads to an event called a first notification. The severity of this first
+notification is a warning. If NotifyInterfaceAdd is set to false, this first
```

Comment:
The severity of this first notification is a OKAY, subsequent notifications are warnings if the state toggles to down